tracex_project/trace_comparator/templates/testing_comparison.html

```diff
@@ -33,7 +33,7 @@ <h3>Patient Journey: {{ patient_journey_name }}</h3>
 
         <form method="post" >
             {% csrf_token %}
-            <button type="submit" class="menu_button" id="progress_button">Start Comparing Against Ground Truth</button>
+            <button type="submit" class="function_button" id="progress_button">Start Comparing Against Ground Truth</button>
```

Comment:
```suggestion
            <button type="submit" class="function_button" id="progress_button" onclick="disableButton()">Start Comparing Against Ground Truth</button>
```

Else the script is useless